1.1/spec/11-Part-08.adoc

```diff
@@ -197,15 +258,30 @@ The `geo:hasSerialization` property is used to connect a geometry with its text-
 
 ```
 geo:hasSerialization a rdf:Property, 
-                       owl:DatatypePropterty;
-    rdfs:isDefinedBy <http://www.opengis.net/spec/geosparql/1.0>; 
+                       owl:DatatypeProperty;
+    rdfs:isDefinedBy <http://www.opengis.net/spec/geosparql/1.1>; 
     rdfs:label "has serialization"@en;
     rdfs:comment "Connects a geometry object with its text-based
                  serialization."@en;
     rdfs:domain geo:Geometry; 
     rdfs:range rdfs:Literal .
 ```
 
+===== Property: geo:inCRS
+
+The `geo:inCRS` property is used to connect a geometry with its text- based serialization (e.g., its WKT serialization).
+
+```
+geo:inCRS a rdf:Property, 
+            owl:ObjectProperty;
+    rdfs:isDefinedBy <http://www.opengis.net/spec/geosparql/1.1>; 
+    rdfs:label "in CRS"@en;
+    rdfs:comment "A Coordinate Reference System, as recorded in a 
```

Comment:
"The coordinate reference system of the coordinates in the representation of the geometry."

Reply:
Fixed by https://github.com/opengeospatial/ogc-geosparql/pull/72/commits/4015f72abbff6c2e4dfe2ef16ca3f97924f13732